src/Commands/Customers/CustomersGroups.php

```diff
@@ -208,7 +208,7 @@ private function groupCustomersTransfer(
         }
 
         // Delete the form group
-        if (self::ACTION_DELETE_FROM_GROUP == (int) $actionAfter && $hasError != 0) {
+        if (self::ACTION_DELETE_FROM_GROUP == (int) $actionAfter && 0 == $hasError) {
```

Comment:
you changed `==` with `!=` it's not a phpstan fix...

Reply:
At first it was `!$hasError`

The error was : 

When I corrected it I was wrong

Reply:
right.
sorry.